src/libraries/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoAbbreviatedMonthNames.cs

```diff
@@ -59,7 +257,7 @@ public void AbbreviatedMonthNames_SetNullValueInValues_ThrowsArgumentNullExcepti
         public static IEnumerable<object[]> AbbreviatedMonthNames_SetInvalidLength_TestData()
         {
             yield return new object[] { new string[] { "Jan" } };
-            yield return new object[] { new string[] { "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec", "", "Additional" } };
+            yield return new object[] { new string[] { "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sept", "Oct", "Nov", "Dec", "", "Additional" } };
```

Comment:
This shouldn't have been changed, it's unintentional and will be reverted. There are no failures because the a test checking if we throw on setting a wrong Length of months, so the actual contents of month names do not matter.